drivers/adc/ad7689/ad7689.c

```diff
@@ -0,0 +1,194 @@
+#include <stdlib.h>
```

Comment:
file header comment missing.

Reply:
Also header comments for each function should be added.

Reply:
you mean doxygen documentation ?

Reply:
yes

Reply:
just added doxygen